### ISSUE TITLE
Validation set

### ISSUE THREAD

**Issue by ap229997** [JudyYe/ihoi#7]
In `models/ihoi.py`, the `val_dataloader` loads the test set and `limit_val_batches` is set to 2. Can you please clarify how is the validation set used for training models on HO3D and MOW datasets? Also, should the `limit_val_batches` be changed to ensure that the validation is done over the entire val set?

**Comment by JudyYe** [comment]
Hi Aditya,
Thank you for reaching out. `limit_val_batch` is effective to limit validation steps in training, not in evaluation (test). The arguments are explained in [pytorch lightning](https://www.pytorchlightning.ai/) document. 
For evaluation, set `args.eval` to True. Code is [here](https://github.com/JudyYe/ihoi/blob/main/models/ihoi.py#L344)

**Comment by ap229997** [comment]
The provided cache does not contain validation set for HO3D and MOW, can you also provide that?
